src/Writer/PageWriter.php

```diff
@@ -144,6 +144,11 @@ public function writePages() // _putpages
 			$this->writer->object();
 			$this->writer->write('<</Type /Page');
 			$this->writer->write('/Parent 1 0 R');
+			if (array_key_exists($n, $this->mpdf->rotations)) {
+				$this->writer->write('/Rotate ' . $this->mpdf->rotations[$n]);
+			} else {
+				$this->writer->write('/Rotate 0');
```

Comment:
Is this 0 rotation necessary or can it be omitted?

Reply:
Omitted.